webpack/utils/helpers.js

```diff
@@ -28,9 +28,9 @@ export const getResponseErrorMsgs = ({ data, actionType } = {}) => {
     const customMessage = getCustomMessage(actionType, data.displayMessage);
     const messages =
       customMessage ||
-      data.errors ||
       data.displayMessage ||
       data.message ||
+      data.errors ||
```

Comment:
`errors` wasn't formatted well in an error which created a `undefined` notification, but it was provided with `displayMessage`. 

It seems that we should be displaying `message` or `displayMessage` in a notification before we dig into the errors object to look for a message.

Reply:
I remembered fixing something similar in #8643... too bad we don't have some sort of standard for this.